06-building-a-program-start-to-finish/setting-up-your-repository/output/README.md

```diff
@@ -0,0 +1,7 @@
+#### Output
+- **Using**: GitHub
+- **Deliver**: the repository for your proposed project and at least one `Pull Request`
+- **With**: *independently*
+
+#### Link to project repository
+- [Inverted-Index Repo](https://github.com/andela-jomadoye/Inverted-index-repo)
```

Comment:
You should improve your ReadMe to accurately describe the project.

[This](https://gist.github.com/q-ode/0b6029f7def28d3006f36d09ea6b849f) should guide you. Please share with your class.